src/components/DataTable/components/Cell/Cell.tsx

```diff
@@ -44,6 +46,7 @@ function Cell({
   const numeric = contentType === 'numeric';
   const className = classNames(
     styles.Cell,
+    styles[`Cell-${variationName('alignment', alignment)}`],
```

Comment:
I'm unfamiliar with how `variationName` works when the variation passed is undefined. Should this be conditional on `alignment` being truthy so it's not given a class name when there's no value set?

Reply:
It can never be undefined - it gets a default value at the start of the function.